examples/clock_benchmark.rb

```diff
@@ -0,0 +1,35 @@
+require 'dynflow'
+require 'benchmark'
+
+class Receiver
+  def initialize(limit, future)
+    @limit = limit
+    @future = future
+    @counter = 0
+  end
+
+  def null
+    @counter += 1
+    @future.success(true) if @counter >= @limit
+  end
+end
+
+def test_case(count)
+  future   = Concurrent.future
```

Comment:
After update of concurrent-ruby, needs to be changed to `future   = Concurrent::Promises.resolvable_future`